docs/source/cli_help/github.rst

```diff
@@ -20,14 +20,16 @@ Creates, checks out and manages GitHub PRs while keeping them reflected in branc
 ``anno-prs``:
     Annotates the branches based on their corresponding GitHub PR numbers and authors.
     Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
-    Equivalent to ``git machete anno --sync-github-prs``.
+    Equivalent to ``git machete anno --sync-github-prs``. Doesn't modify existing ``git machete traverse`` branch qualifiers.
```

Comment:
Hmmm TBH I'm not sure... shouldn't `github anno-prs` (and `anno -H`) also fill up the qualifiers for someone else's PRs? 🤔 TBH I'd rather expect that

Reply:
Also: `the existing`

Reply:
Alright, i wasn't 100% convinced but if it's expected i will add it